### DISCUSSION TITLE
Image output path problem

### DISCUSSION THREAD

**Discussion by nolanjj** [quarto-dev/quarto-cli#2775] [Q&A]
such as render 11.qmd Image will save in 11_files folder。I want  Image save in 11 folder

I open D:\Program Files\RStudio\bin\quarto\bin\quarto.js

![image](https://user-images.githubusercontent.com/35267241/194807254-3ca4ae4f-c7ec-41e0-9618-231c7e05e07e.png)

I remove "_files"  

But it didn't work

**Comment by mcanouil** [comment]
Related to the initial question in #2767

You can't simply edit one "random" file like that (if you really wanted to do that you would need to work on the source code, not the compiled one, _i.e._, <https://github.com/quarto-dev/quarto-cli>), see partly why on my reply to your other thread: https://github.com/quarto-dev/quarto-cli/discussions/2767#discussioncomment-3837066

**Comment by dragonstyle** [comment]
The rest of the Quarto system has enough of a dependency upon the `<input>_files` dir being used for output dependencies that we don't currently support changing this (and it will be a big change if we ever do).

**Reply by nolanjj** [reply]
> The rest of the Quarto system has enough of a dependency upon the `<input>_files` dir being used for output dependencies that we don't currently support changing this (and it will be a big change if we ever do).

thank you!! finally I found a way to to achieve my purpose

I run
`rename 's/_files//' *_files
`ls *.md |while read id;do(cat $id | sed -i 's/_files//' $id );done`

 then removes all "_files" suffixes

So I can upload the .md and fig to my hexo blog and the fig can be displayed normally
